apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -2036,10 +2036,13 @@ else:
         });
 
         $("#fileUploader").on('fb:updatePath', function (e, options) {
-          const uploadingToOzone = self.currentPath().startsWith("ofs://");          
+          const uploadingToOzone = self.currentPath().startsWith("ofs://");      
           if (uploadingToOzone && UPLOAD_CHUNK_SIZE) {
-            uploader.setOption('sizeLimit', UPLOAD_CHUNK_SIZE);
-          }  
+            uploader.setOption('sizeLimit', Math.min(UPLOAD_CHUNK_SIZE, UPLOAD_SIZE_LIMIT));
+          }
+          else if (UPLOAD_SIZE_LIMIT !== -1) {
+            uploader.setOption('sizeLimit', UPLOAD_SIZE_LIMIT);
+          }
```

Comment:
What happens if `uploadingToOzone` is true but `UPLOAD_CHUNK_SIZE` is false? Is that a working scenario or must we warn in that case?

Reply:
As far as know UPLOD_CHUNK_SIZE will never be false. Default value is 64MB. If user wants to upload a file then I don't think user will set it to zero. Let me know if we still need to add warning.

Reply:
Done
Thank you

Reply:
@bjornalm  all the changes are in now, can you please check it.